frontend/app/views/fragments/bundle/bundleOfferingsA.scala.html

```diff
@@ -5,207 +5,210 @@
+                                        <h1>Sixday</h1>
+                                        <p>Monday - Saturday</p>
+                                    </div><div class="bundle-offering-a__print-option__price">
+                                @bundleVariant.prettyMonthlyPrice(SixDay)
                                 </div>
-                                <div class="bundle-offering-a__print-option__id">seven-day</div>
-                            </li>
-                        </ul>
-                        @fragments.bundle.bundleButton(bundleVariant.prettyMonthlyPrice(Saturday), routes.Bundle.thankYou(bundleVariant, "saturday").url, Option("js-print-button"), returnUrl = Some(java.net.URLEncoder.encode(returnUrl.getOrElse("https://www.theguardian.com"), "utf-8")))
+                                    <div class="bundle-offering-a__print-option__id">six-day</div>
+                                </li>
+                                <li class="bundle-offering-a__subscribe__offer__print-options__option">
+                                    <div class="bundle-offering-a__print-option__description">
+                                        <h1>Everyday</h1>
+                                        <p>Monday - Sunday</p>
+                                    </div><div class="bundle-offering-a__print-option__price">
+                                @bundleVariant.prettyMonthlyPrice(SevenDay)
+                                </div>
+                                    <div class="bundle-offering-a__print-option__id">seven-day</div>
+                                </li>
+                            </ul>
+                            @fragments.bundle.bundleButton(bundleVariant.prettyMonthlyPrice(Saturday), routes.Bundle.thankYou(bundleVariant, "saturday").url, Option("js-print-button"), returnUrl = Some(java.net.URLEncoder.encode(returnUrl.getOrElse("https://www.theguardian.com"), "utf-8")))
+                        </div>
                     </div>
                 </div>
-            </div>
+            }
         </div>
+        @if(bundleVariant.hasContributions) {
+            <div class="bundle-offering-a__give">
+                <div class="bundle-offering-a__give__content">
+                    <h1>Support through giving</h1>
+                    <div class="bundle-offering-a__give__content__body">
+                        <p class="bundle-offering-a__give__content__body__description">
+                            Fund our journalism on your own terms – either a regular or one-off payment. Every penny goes to supporting our fearless, in-depth reporting and keeping our website and apps open to all.</p>
+                        <div class="bundle-offering-a__give__button">
+                        @fragments.bundle.bundleButton("Make a contribution", s"https://support.theguardian.com/uk?INTCMP=${bundleVariant.testId}", className = Option("js-commit-button"))
```

Comment:
Does the domain need to be hardcoded here since they're currently on the same site? Would `/uk?INTCMP=${bundleVariant.testId}` work?